symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/user/OboUserService.java

```diff
@@ -20,121 +20,120 @@
 
 import javax.annotation.Nullable;
 
+/**
+ * Service class for managing users. This exposes OBO-enabled endpoints only.
+ */
 @API(status = API.Status.INTERNAL)
-class OboUserService {
+public class OboUserService {
 
   protected final UserApi userApi;
   protected final UsersApi usersApi;
+  protected final AuthSession authSession;
   protected final RetryWithRecoveryBuilder retryBuilder;
 
-  protected OboUserService(UserApi userApi, UsersApi usersApi, RetryWithRecoveryBuilder retryBuilder) {
+  public OboUserService(UserApi userApi, UsersApi usersApi,AuthSession authSession, RetryWithRecoveryBuilder retryBuilder) {
     this.userApi = userApi;
     this.usersApi = usersApi;
+    this.authSession = authSession;
     this.retryBuilder = retryBuilder;
   }
 
   /**
    * {@link UserService#searchUserByIds(List, Boolean)}
    *
-   * @param authSession Bot session or Obo Session
    * @param uidList     List of user ids
    * @param local       If true then a local DB search will be performed and only local pod users will be
    *                    returned. If absent or false then a directory search will be performed and users
    *                    from other pods who are visible to the calling user will also be returned.
    * @return Users found by user ids
    * @see <a href="https://developers.symphony.com/restapi/reference#users-lookup-v3">Users Lookup V3</a>
    */
-  public List<UserV2> searchUserByIds(@NonNull AuthSession authSession, @NonNull List<Long> uidList,
+  public List<UserV2> searchUserByIds(@NonNull List<Long> uidList,
```

Comment:
Just realized that `@NonNull` is the Lombok one, we should use `@javax.annotation.Nonnull` instead